src/store/modules/user/index.ts

```diff
@@ -12,6 +12,7 @@ const userModule: Module<UserState, RootState> = {
       current: null,
       currentFacility: {},
       instanceUrl: '',
+      fieldMappings: []
```

Comment:
We could have it as object with key as mappingPrefId and value as object having mappingPrefId, mappingPrefName, mappingPrefTypeEnumId and mappingPrefValue

Reply:
Created it as an object with key as mapping name and value as mapping fields.